worlds/ladx/__init__.py

```diff
@@ -378,60 +379,26 @@ def priority(item):
     # Tries to associate an icon from another game with an icon we have
     def guess_icon_for_other_world(self, other):
         if not self.name_cache:
-            forbidden = [
-                "TRADING",
-                "ITEM",
-                "BAD",
-                "SINGLE",
-                "UPGRADE",
-                "BLUE",
-                "RED",
-                "NOTHING",
-                "MESSAGE",
-            ]
             for item in ladxr_item_to_la_item_name.keys():
                 self.name_cache[item] = item
                 splits = item.split("_")
                 self.name_cache["".join(splits)] = item
                 if 'RUPEES' in splits:

```

Comment:
Can you change this to "RUPEE"? Currently "1 RUPEE" is turned into a mystery item